src/prisma/schema/event.prisma

```diff
@@ -0,0 +1,61 @@
+enum EventCanView {
+  CAN_REGISTER
+  ALL
+}
+
+model Event {
+  id             Int             @id @default(autoincrement())
+  name           String
+  omegaOrder     OmegaOrder      @relation(fields: [order], references: [order])
+  order          Int
+  paragraph      CmsParagraph    @relation(fields: [cmsParagraphId], references: [id])
+  cmsParagraphId Int             @unique
+  coverImage     CmsImage        @relation(fields: [coverImageId], references: [id])
+  coverImageId   Int
+  EventTagEvent  EventTagEvent[]
+
+  eventStart         DateTime
+  eventEnd           DateTime
+  canBeViewdBy       EventCanView
+  hostedByCommitee   Committee?   @relation(fields: [hostedByCommiteeId], references: [id])
+  hostedByCommiteeId Int?
+
+  takesRegistration Boolean
+  places            Int                 @default(0)
+  registrationStart DateTime
+  registrationEnd   DateTime
+  EventRegistration EventRegistration[]
```

Comment:
Not a big think, but it annoys me that the prisma extension PascalCase for automatically added relations when it should be camelCase